spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/config/GoogleConfigPropertySourceLocatorTest.java

```diff
@@ -79,10 +79,10 @@ public void locateReturnsMapPropertySource() throws Exception {
 				this.projectIdProvider, this.credentialsProvider, this.gcpConfigProperties));
 		doReturn(googleConfigEnvironment).when(this.googleConfigPropertySourceLocator).getRemoteEnvironment();
 		PropertySource<?> propertySource = this.googleConfigPropertySourceLocator.locate(new StandardEnvironment());
-		assertEquals(propertySource.getName(), "spring-cloud-gcp");
-		assertEquals(propertySource.getProperty("property-int"), 10);
-		assertEquals(propertySource.getProperty("property-bool"), true);
-		assertEquals("projectid", this.googleConfigPropertySourceLocator.getProjectId());
+		assertThat(propertySource.getName()).isEqualTo("spring-cloud-gcp");
+		assertThat(propertySource.getProperty("property-int")).isEqualTo(10);
+		assertThat(propertySource.getProperty("property-bool")).isEqualTo(true);
```

Comment:
is there a isTrue() ?

Reply:
This one is tricky since `propertySource.getProperty("property-bool")` returns an object so I couldn't use isTrue directly; would have to cast the input to a boolean first. I figure `isEqualTo(true)` was preferable to a cast to boolean + isTrue.